### ISSUE TITLE
netcdf tools not working

### ISSUE THREAD

**Issue by sebastianWI** [ContinuumIO/anaconda-issues#1346]
conda install netcdf also installs a couple of command line programs (e.g. ncdump, nccopy etc). They do not work properly. With all netcdf-files, they produce *** ncdump:  ncopen failed on file.nc
I experienced the problem both on windows 10, windows server 2008 and Ubuntu 16.04. If I install the netcdf tools manually (e.g. via the windows installer), everything works fine.

Note tha inside python, the netcdf library installed with conda works without problems. Only the command line tools do not work. As adding the anaconda bin directory to PATH overwrites the path to a manuall netcdf installs, this is still quite a problem. I think the tools should be either fixed or removed.

**Comment by jhamman** [comment]
I've also seen this issue. I usually end up deleting the command line tools added to my `bin`.

**Comment by silentquasar** [comment]
I've run up against the same issue. I'm using Anaconda3 4.3.0 x64 on CentOS 7.

Here's what I learned. When I `conda install netcdf4`, it installs the hdf4 package, which has its own versions of ncgen and ncdump that are not useful at all for NetCDF4 files and have a very different syntax. These rely on an earlier version of libjpeg than is in Anaconda3 4.3.0 (I was able to get them to work with `conda install jpeg=8d`).

It would be expected that when libnetcdf is installed that the NetCDF utilities would also come along for the ride and override the hdf4 ones. Actually in conda-forge they have resolved this by renaming the hdf4 ncgen and ncdump utilities to h4_ncgen and h4_ncdump:. https://github.com/conda-forge/hdf4-feedstock/issues/5.

This has also been proposed as a change in the hdf group: https://lists.hdfgroup.org/pipermail/hdf-forum_lists.hdfgroup.org/2013-November/007300.html
